### ISSUE TITLE
Issues with Chrome

### ISSUE THREAD

**Issue by agzam** [ch11ng/exwm#487]
Anyone else is experiencing strange bug with Google Chrome? Sometimes it simply won't switch back to Chrome buffer - it hangs. I have a feeling this is strictly Chrome related problem on Arch. Firefox seems to be working fine. I have tried different things - switching to chrome-dev, starting it via `dbus-launch`. This is driving me crazy.

**Comment by medranocalvo** [comment]
Hello @agzam.  That seems weird.  Please, enable debug logging and try to reproduce:

<kbd>M-x exwm-debug</kbd>

The debug log will be in a buffer named `*XELB-DEBUG*`.  When the failure occurs, switch to that buffer and paste here some relevant part of the tail.  Watch out for and remove any private information (keypresses, window titles, ...).

**Comment by QiangF** [comment]
Are you using your own exwm-edit? exwm-send-fake-kay may cause the problem. You can try with xdotool

          (start-process-shell-command "xdo paste" nil (concat "xdotool key --delay 40 " the-paste-key))

**Comment by ch11ng** [comment]
> Sometimes it simply won't switch back to Chrome buffer - it hangs.

Which hangs, Chrome or Emacs? Could you elaborate the problem?

**Comment by agzam** [comment]
@ch11ng Chrome hangs. Once again - my apologies. This issue does not seem to be anyhow related to EXWM, I think this is a bug in Chrome or in general, an issue in Arch. I asked in `/r/arch` reddit, I search for similar issues, but still haven't found a solution or was able to pinpoint the exact cause of this.
I've created this issue out of desperation hoping maybe somebody has similar problems or clues. 
t might be related to dbus, I dunno. I decided to try starting the EXWM session without dbus and see how it goes.

Earlier I experienced a different issue with Chrome in xfce - somehow enabling sync would make it unable to start again (after exiting). Sad - it looks like Chrome becoming more and more unstable in Linux.

Anyway, it's probably best if I close this issue for now. If it continues to misbehave I guess I'm gonna have to permanently switch to a different browser, because I'm definitely not leaving EXWM - I love this thing. It has certain quirks but it is getting better everyday.

Thank you all for your help!

**Comment by agzam** [comment]
> Are you using your own exwm-edit?
@QiangF 

no, I don't think this related to exwm-edit at all, but I'll keep an eye. exwm-edit may cause different issues, but I don't think this is one of them. Please don't hesitate to let me know if you encounter any problems with exwm-edit. Thank you!

**Comment by medranocalvo** [comment]
@agzam, this sounds similar to https://github.com/ch11ng/exwm/issues/210#issuecomment-350044271.  Try whether any of the suggestions linked from my comment https://github.com/ch11ng/exwm/issues/210#issuecomment-427869226 helps in your case.

**Comment by agzam** [comment]
I turned on exwm-debug. It seems throwing a bunch of `exwm--on-PropertyNotify: Unhandled PropertyNotify: WM_STATE(349)` kind of errors

**Comment by medranocalvo** [comment]
Hello @agzam.  Despite the alarming sound, `exwm--on-PropertyNotify: Unhandled PropertyNotify: WM_STATE(349)` is innocuous.  I'm closing this ticket in favour of #27, as it seems the exact same issue.

**Comment by agzam** [comment]
Sorry, it took a while. I just wanted to make sure it is not dying anymore. So I googled and asked, and finally someone on reddit mentioned something about Google recently adding native notifications support to Chromium. And then I went to `chrome:://flags` and turned off **everything** that had anything to do with notifications. It feels much more stable now. To be honest - I'm very happy to find out that this has nothing to do with EXWM and I apologize for freaking out and being dramatic. It was extremely annoying though.
